code/modules/reagents/reagents/toxin.dm

```diff
@@ -501,7 +501,6 @@
 	scannable = TRUE
 
 /datum/reagent/toxin/xeno_growthtoxin/on_mob_life(mob/living/L)
-	L.reagent_pain_modifier += PAIN_REDUCTION_VERY_HEAVY
```

Comment:
i dunno about this, it makes sense this exists since it purges tramadol and chems like that, making you basically stay without painkillers which are veeeery important.

Reply:
This is in anticipation of stamina loss being nerfed with perma stun being removed

Reply:
is that the same as tramadol or better ? i'd be okay with it doing same as tramadol or slightly less, seeing how painkillers are really neccessary and removing them fast from a marine is a lot

Reply:
We literally test ran this removal with the Defiler TM and it was fine; the main complaint wasn't pre-med depletion (that actually didn't come up at all), but perma stunning from neuro spam combined with Defile.

Reply:
might just be something people dont notice, also more seeing the defiler thing makes every slash of his inject this, its literally remove painkillers and all in one click, painkillers are very important on game and getting without is veeery veery bad

Reply:
So here's my offer: I'm keeping the change to the toxin, but it won't inject unless the target has a larva.